source/shared/net/desktop-wallet/update-baker-keys.rst

```diff
@@ -0,0 +1,116 @@
+.. _update-baker-keys:
+
+=======================================
+Update baker keys in the Desktop Wallet
+=======================================
+
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 1
+
+If for some reason you lose the public baker keys, we recommend that you update the keys and configure the baker node with the new keys.
+
+Prerequisites
+=============
+
+-  Set up an account as a baker account.
+-  Set up a Ledger hardware wallet.
+
+Select an account
+=================
+
+#. Go to **Multi Signature Transactions**, and select **Make new proposal**, and then select **Update Baker Keys**.
+
+#. Select the **Identity** that's associated with the baker account whose keys you want to update, and then select **Continue**.
+
+#. Select the **Account** whose baker keys you want to update, and then select **Continue**. Only baker accounts are listed.
+
+#. Set an expiry date and time for your proposal. You must submit the proposal to the chain within the last 2 hours up to the expiry date. Consider this when you set the expiry time so that the co-signers can return their signatures in time. Select **Generate keys**.
+
+   The baker keys are generated and you can view the transaction details in the left pane. You can see the identity, the account, and the expiry time of the transaction. You can also see the public baker keys.
+
+#. You have to export the baker credentials so that you can restart the node with the new baker keys. Select **Export baker credentials**. If you're on Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save**.and navigate to the place on your computer where you want to save the file.
```

Comment:
```suggestion
#. You have to export the baker credentials so that you can restart the node with the new baker keys. Select **Export baker credentials**. If you're on Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save** and navigate to the place on your computer where you want to save the file.
```